internal/resources/resource_volume.go

```diff
@@ -36,6 +37,9 @@ const (
 	vID          = "id"
 	vDiscoveryIP = "discovery_ip"
 	vTargetIQN   = "target_iqn"
+
+	GBToGiBConversion float64 = 0.931323
```

Comment:
Not sure why this is rounded to 0.931323. Math says it's 0.931322574615478515625, so should be 0.931322. Google shows a few places doing 0.931323. Not sure I agree, but probabaly doesn't matter.

Reply:
0.931322574615478515625 rounded to 6 places is 0.931323